index.html

```diff
@@ -97,7 +101,7 @@ <h1 class="work-for-you">РАБОТЫ, КОТОРЫЕ МЫ МОЖЕМ ВЫПОЛ
         </div>
     </div>
     <div class="footer">
-        <img src="./img/plumber-footer.webp" alt="logo">
+        <img class="footer-logo" src="favicon.ico" alt="logo" style: width="100px">
```

Comment:
`favicon.ico` must be used just as favicon, not the image. Please use `./img/plumber-footer.webp` as you did before, not `favicon.ico`. And add `favicon.ico` as you've done with the rest favicons in `head` tag:
`<link rel="icon" type="image/x-icon" href="/favicon/favicon.ico">`